target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -52,6 +52,17 @@ netgear,r6850|\
 netgear,wndr3700-v5)
 	ucidef_set_led_switch "wan" "wan" "$boardname:green:wan" "switch0" "0x10"
 	;;
+netgear,r6800)
+	ucidef_set_led_switch "wan" "WAN" "$boardname:white:wan" "switch0" "0x10"
+	ucidef_set_led_switch "lan1" "LAN1" "$boardname:white:lan1" "switch0" "0x08"
+	ucidef_set_led_switch "lan2" "LAN2" "$boardname:white:lan2" "switch0" "0x04"
+	ucidef_set_led_switch "lan3" "LAN3" "$boardname:white:lan3" "switch0" "0x02"
+	ucidef_set_led_switch "lan4" "LAN4" "$boardname:white:lan4" "switch0" "0x01"
+	ucidef_set_led_netdev "wlan2g" "WiFi 2.4GHz" "$boardname:white:wlan2g" "wlan0"
+	ucidef_set_led_netdev "wlan5g" "WiFi 5GHz" "$boardname:white:wlan5g" "wlan1"
+	ucidef_set_led_usbport "usb2" "USB 2.0" "$boardname:white:usb2" "usb1-port2"
+	ucidef_set_led_usbport "usb3" "USB 3.0" "$boardname:white:usb3" "usb1-port1"
```

Comment:
WiFi and USB leds should be configurable in DTS instead.

Reply:
Done.